authorship/Function_file.py

```diff
@@ -134,9 +134,12 @@ def compile_data_CGN(string):
 
 def get_frequent_words(speakers, n):
```

Comment:
maybe add some docstr here to describe what this function does?

Reply:
done